check_io.py

```diff
@@ -1,10 +1,9 @@
 import pandas as pd
 import sys
 
-s1 = sys.argv
-csvfile = s1.pop()
+rd = sys.argv
+csvfile = rd.pop()
```

Comment:
use camelCasing

Reply:
try to get more values in arguments like number of itration, device type, sensor connected or not, and csv name